Neos.Error.Messages/Classes/Message.php

```diff
@@ -57,15 +57,15 @@ class Message
      * Constructs this error
      *
      * @param string $message An english error message which is used if no other error message can be resolved
-     * @param integer $code A unique error code
+     * @param integer|null $code A unique error code
      * @param array $arguments Array of arguments to be replaced in message
-     * @param string $title optional title for the message
+     * @param string|null $title optional title for the message
      * @api
      */
-    public function __construct(string $message, int $code = null, array $arguments = [], string $title = '')
+    public function __construct(string $message, int $code = null, array $arguments = [], $title = '')
     {
         $this->message = $message;
-        $this->code = (int)$code;
+        $this->code = $code;
```

Comment:
This was actually a non b/c change, since `code` really defaulted to `null` before.